src/css/betterttv-dark.css

```diff
@@ -3807,6 +3807,10 @@ button.follow-button, .subscription-modal__right button, .subscription-modal__le
     color: #d3d3d3;
 }
 
+.video-chat__message-author span {
+	color: inherit;
```

Comment:
we use spaces not tabs

Reply:
That's strange, my editor should not be set to tabs... I use spaces.

Will fix now.